torchmetrics/classification/accuracy.py

```diff
@@ -103,15 +163,31 @@ class Accuracy(Metric):
 
     def __init__(
         self,
+        num_classes: Optional[int] = None,
         threshold: float = 0.5,
```

Comment:
```suggestion
        threshold: float = 0.5,
        num_classes: Optional[int] = None,
```
anyway this adding args in middle can be breaking API change for users passing args without key word...